.devcontainer/updateContent.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+apt update
+apt install libgdal-dev
```

Comment:
This prevents mac users from using this codespace, barring some magic I am unaware of that would make this work.

Reply:
I don't understand codespaces, I thought it was running in a container.

I guess we can use `micromamba` to build a cross-platform thing, but I'll leave that alone for now

Reply:
I think it is and I am wrong. Feel free to reopen if you want.